src/build/build_helpers.jl

```diff
@@ -61,6 +61,8 @@ function swap_our_param_with_val(expr)
             else
                 return a.benchmark[x.subindex]
             end
+        elseif x isa JuMP.VariableRef
+            return MPSGE.Complementarity.result_value(x)
```

Comment:
Adding this enabled the first part of the ConsValue expression to evaluate to a Number